addons/website_event_track/models/event_track.py

```diff
@@ -163,6 +163,14 @@ def _compute_website_url(self):
             if track.id:
                 track.website_url = '/event/%s/track/%s' % (slug(track.event_id), slug(track))
 
+    @api.depends('stage_id')
+    def _compute_is_visible(self):
```

Comment:
Not sure this is always valid ... shoudln't managers have a way to prepare their agenda ?